### ISSUE TITLE
Mer Laboratorio 2

### ISSUE THREAD

**Issue by AlexisVM** [TechNix-Company/BasesDeDatos#8]
Hola! revisé su entrega para el laboratorio 2, todo se ve muy bien. Podrían para modelos posteriores indagar más en las relaciones, por ejemplo a mi parecer la relación entre pago y reservación está ambigua y creo que el tipo de pago podría ser simplemente otro  atributo de la entidad pago,  no se si aplique algo así en otro lado.

**Comment by PValenciac** [comment]
Sí, tienes toda la razón Alexis. Recuerdo que los pusimos con la intención de tener todos los tipos de pago en un catálogo. Muchas gracias por la observación.
